core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -601,6 +601,12 @@ package object config {
       .booleanConf
       .createWithDefault(false)
 
+  private[spark] val DYN_ALLOCATION_STREAMING_ENABLED =
+    ConfigBuilder("spark.dynamicAllocation.streaming.enabled")
```

Comment:
how does this interact with spark.dynamicAllocation.enabled	?  I think it needs to be clear to the user and docuemnted well.

you also need to update the configuration.md documentation to show this publicly

Reply:
Will update all `configuration.md` and `job-scheduling.md` in the next commit

Reply:
Yes need to clarify whether it can be enabled separately or not.